src/test/examples/sparse-array.html

```diff
@@ -0,0 +1,20 @@
+<html>
+  <head>
+    <title>Debugger Statements</title>
+  </head>
+
+  <body>
+    <script>
+      debugger;
+
+      localStorage[1] = "a test";
+      localStorage[2] = "a";
+      localStorage[66261280] = "another test";
+      let data = JSON.parse(JSON.stringify(localStorage));
+
```

Comment:
Debugger statement should be put here. (if data doesnt have a value it wont crash)